README.md

```diff
@@ -90,33 +62,50 @@ After you finish this process, you should be able
 to type `python --version` and get `Python 3.7.2` or other version after
 `Python 3.6.8` in your terminal.
 
-## Initial Setup
+As a Python 3 program, GatorGrouper relies on
+[Pipenv](https://github.com/pypa/pipenv) for the installation of the libraries
+on which it depends and the creation of the virtual environments in which it
+runs. To install GatorGrader, you should first follow Pipenv's installation
+instructions. You should also ensure that you have installed Git on your
+computer and that you can run Git commands in a terminal window. Then, you can
+type the following command in your terminal window to clone GatorGrader's GitHub
```

Comment:
`GatorGrader` is mentioned multiple times in this area.

Reply:
This should be resolved.